### PR TITLE
Remove `echo chat` and `leaderboard`

### PR DESCRIPTION
**Pull request by Kendiya242**
I officially remove eco chat and leaderboard from the nav-bar.

### CODE REVIEW

**Review by dinaolmelak — APPROVED** [review]
ok 👍🏽

**Comment by dinaolmelak** [comment]
Dont forget to comment out the imports for the `Eco chat` and  `Leaderboard` @Kendiya242

File: src/component/navbar/Navbar.jsx
Comment (dinaolmelak) [inline src/component/navbar/Navbar.jsx:1]: no worries, @Kendiya242 , `Navbar.jsx` does not have those imports so disregard the earlier comment.

Comment (Kendiya242) [reply src/component/navbar/Navbar.jsx:1]: ok I am going to merge this.

**Comment by dinaolmelak** [comment]
Fine by me @Kendiya242

**Pull request merged**